packages/ui/src/forum/queries/forum.graphql

```diff
@@ -204,7 +207,7 @@ query GetForumPosts($where: ForumPostWhereInput!, $orderBy: [ForumPostOrderByInp
 }
 
 query GetForumPostEdits($id: ID!) {
-  edits: postTextUpdatedEvents(where: { post: { id_eq: $id } }) {
+  edits: postTextUpdatedEvents(where: { post: { id_eq: $id } }, orderBy: [createdAt_ASC]) {
```

Comment:
I'd assume the most recent edit would go to the top instead, as it's the most relevant one. Not sure what the design is though